google/resource_compute_image.go

```diff
@@ -195,3 +213,9 @@ func resourceComputeImageDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceComputeImageImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	d.Set("create_timeout", computeImageCreateTimeoutDefault)
```

Comment:
Added a comment explaining why.

Another option would be to set the `create_timeout` in the read method if it is not set but I prefer not cluttering the read method logic with logic specific to import.